index.html

```diff
@@ -9,13 +9,17 @@
 	<meta name="author" content="">
 	<link rel="icon" href="../../favicon.ico">
```

Comment:
Actually this is because the icon file "favicon.ico" is missing, will download the icon file and redirect to the correct path